### ISSUE TITLE
Umbraco.TinyMCEv3 Rich Text Editor Internal Links Not Working

### ISSUE THREAD

**Issue by JMMM77** [Jumoo/uSyncMigrations#139]
uSync.Migrations: `4.0.0-phase.2.1`
uSync.Cms: `10.3.3`
Hi,

I noticed that the RTE internal links were not working, I think its because the localLink is missing umb://document/.

I was thinking about creating my own custom migratory to fix this but I couldn't find an example that updates the value of a property in the [MyMigrationProfile.cs](https://github.com/Jumoo/uSyncMigrations/blob/main/MyMigrations/MyMigrationProfile.cs).

Before:
![image](https://github.com/Jumoo/uSyncMigrations/assets/91952323/482d426f-57b5-4e55-ae35-67cd8dd6576f)

After:
![image](https://github.com/Jumoo/uSyncMigrations/assets/91952323/3c8e5e3c-d5fe-4d93-a4f5-df28b3c860bf)

As a workaround I ran the below SQL Script to get all the pages that are using the Umbraco.TinyMCEv3 and is using "localLink:".

``` sql
  Select upd.id as 'umbracoPropertyData id'
  , propertyTypeId
  , udt.propertyEditorAlias
  , cpt.Alias as 'Property Type Alias'
  , ucv.text as 'Node Name'
  , ucv.[current]
  , published
  , varcharValue
  , textValue
	from umbracoPropertyData upd 
	INNER JOIN umbracoContentVersion ucv ON upd.versionId = ucv.Id 
	INNER JOIN cmsPropertyType as cpt on cpt.id = upd.propertyTypeId
	INNER JOIN umbracoDataType as udt on cpt.dataTypeId = udt.nodeId
	INNER JOIN umbracoDocument as ud on ud.nodeId = ucv.nodeId
	WHERE  (ucv.[current] = 1 and propertyEditorAlias='Umbraco.TinyMCE' and textValue like '%href="/{localLink:%') 
```

Please let me know if there is a better way to go about this.

**Comment by KevinJump** [comment]
Hi, 

I think the [RichTextBoxMigrator](https://github.com/Jumoo/uSyncMigrations/blob/main/uSync.Migrations/Migrators/Core/RichTextBoxMigrator.cs) will need expanding (at the moment it just passes values through).

overwriting the `GetContentValue` method from the base class would enable the migrator to find the links (via regex) and update them.

**Comment by bielu** [comment]
@KevinJump I think this one was already addressed 🤔

**Comment by AllanMT** [comment]
Yes, I had this problem earlier in the year but it seemed fixed with the latest version on November 14th
